legacy-fabric-command-api-v2/1.6.4/src/main/java/net/legacyfabric/fabric/api/command/v2/lib/sponge/CommandCallable.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2020 - 2024 Legacy Fabric
+ * Copyright (c) 2016 - 2022 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
MIT license - this is from SpongeAPI

Reply:
wait i just realized all the spongeapi headers got deleted
see https://github.com/Legacy-Fabric/fabric/blob/38f2c1406b937bc4933d6b2a375a2cfff1486b59/modules/server/sponge-commands/src/main/java/net/legacyfabric/fabric/api/command/v2/lib/sponge/args/parsing/RawStringInputTokenizer.java#L2 and https://github.com/Legacy-Fabric/fabric/blob/38f2c1406b937bc4933d6b2a375a2cfff1486b59/build.gradle#L141

Reply:
Covered by #174